rice_irrigation.csv

```diff
@@ -0,0 +1,46 @@
+loss,elapsed_budget,elapsed_time,error,pseudotime,num_objectives,info/sub-optim,seed,symmetry,n_iterations,function_class,parametrization,dimension,budget,num_workers,batch_mode,optimizer_name
```

Comment:
we should avoid pushing data on the repo. Only code should be versioned.

Please could you remove csv and png files from your branch?